homeassistant/components/climate/dyson.py

```diff
@@ -187,9 +186,9 @@ def set_operation_mode(self, operation_mode):
     @property
     def min_temp(self):
         """Return the minimum temperature."""
-        return convert_temperature(1, TEMP_CELSIUS, self.temperature_unit)
+        return 1
```

Comment:
Not sure if I should make these as constants or not. This weren't defined in the library as constants (but enforced with a check against `if temperature < 1 or temperature > 37`) so I cannot import them. It just seems redundant to define them as constants in this component alone as they are only used once.